lib/coffeescript/nodes.js

```diff
@@ -2732,7 +2740,7 @@
           }
           rite = new Call(rite, this.args);
           rite.isNew = this.isNew;
-          left = new Literal(`typeof ${left.compile(o)} === "function"`);
+          left = new Literal(`typeof ${left.compile(o)} === \"function\"`);
```

Comment:
This is slightly interesting (from `lib/coffeescript/` being recompiled using the heredoc -> template literal change):

Escaped quotes from the original string are now preserved - before, they were getting removed because we escaped quotes when storing the quoted version of the string chunk (as the `StringLiteral`'s `@value`) and then unescaped all quotes (whether originally escaped in the source or escaped by us) when `unquote()`ing the string chunk

This seems fine (I don't think there's any reason we'd need/want to strip unnecessary escapes (that are present in the original source) from the output)?